packages/client/hmi-client/src/workflow/ops/calibrate-ciemss/tera-calibrate-ciemss.vue

```diff
@@ -31,6 +36,18 @@
 								/>
 							</template>
 						</Column>
+						<Column field="deleteRow">
+							<template #header>
+								<span class="column-header"></span>
+							</template>
+							<template #body="{ index }">
+								<Button
+									class="p-button-sm p-button-text"
+									label="Delete Row"
```

Comment:
The interface looks a tad cramped in your video, maybe we can just shorten this to "delete" instead of "delete row"?